rocketpy/Flight.py

```diff
@@ -527,6 +529,7 @@ def __init__(
         atol=6 * [1e-3] + 4 * [1e-6] + 3 * [1e-3],
         timeOvershoot=True,
         verbose=False,
+        name="",
```

Comment:
To make it more standardized with what is in AeroSurfaces I think the default for the `name` parameter should be `name = "Flight"`